package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/event-sdk",
-  "version": "8.8.0",
+  "version": "8.8.1",
```

Comment:
v8.8.0 is fine. We have not made an official release yet.

No idea what @vgenev is doing on 8.8.1-snapshot :D

Reply:
Sure. I'll remove that last commit